functions/src/api/v1/invite/inviteParticipants.ts

```diff
@@ -59,16 +60,40 @@ import { increment } from '../../../firebase/firebase'
  *       412:
  *         description: authorization header wrong format
  */
-export const inviteParticipants = wrap(async (req: Request, res: Response) => {
-    const roomId = req.params.roomId
-    const userId: UID = res.locals.uid
-    const room = await assertRightToEditRoom(roomId, userId)
+export const inviteParticipantsRoute = wrap(
+    async (req: Request, res: Response) => {
+        const roomId = req.params.roomId
+        const userId: UID = res.locals.uid
+        const { hostName, destinations } = req.body
+
+        const invitationsSent = await inviteParticipant({
+            roomId,
+            userId,
+            hostName,
+            destinationsParameter: destinations,
```

Comment:
pls consider `JSON.parsing` the destinations here, so that you can directly forward the `destination` parameter to `inviteParticipant` function

Reply:
the idea was to regroupe the field format at a single place to allow easier update in the future (like supporting full json payload)

Reply:
does this mean that:

```js
const invitationsSent = await inviteParticipant({
  roomId,
  userId,
  hostName,
  destinations
})  
```

would not fit your purpose ?